src/Sylius/Component/Core/Model/OrderItemUnit.php

```diff
@@ -135,4 +135,20 @@ public function setShippingState($state)
     {
         $this->shippingState = $state;
     }
+
+    /**
+     * Returns sum of neutral and non neutral tax adjustments.
+     *
+     * {@inheritdoc}
+     */
+    public function getTaxTotal()
```

Comment:
As per our discussion, I recommend creating a `getNeutrallyInclusiveAdjustmentsTotalRecursively` method at the Order level to cater for a wider scope of possibilities and then add `getTaxTotal()` methods which wraps these at each level.